ssg/build_cpe.py

```diff
@@ -87,6 +88,9 @@ def get_cpe(self, ref):
             if self._is_name(ref):
                 return self.cpes_by_name[ref]
             else:
+                # in case we get parametrized item in form ref[parameter]
+                # get only the ref part
+                ref = pkg_resources.Requirement.parse(ref).project_name
```

Comment:
I think It would be better to add some more abstraction here and "hide" the Requirement class. You can extract this line to a new separate function with a good name.